osier/technology.py

```diff
@@ -173,6 +208,9 @@ class Technology(object):
     default_time_units : str or :class:`unyt.unit_object.Unit`
         An optional parameter, specifies the units
         for time. Default is hours [hr].
+    default_mass_units : str or :class:`unyt.unit_object.Unit`
```

Comment:
I don't think the `:class:` specifier is necessary when specifying the type.

Reply:
It's not really for the casual source code reader, it helps sphinx format the docstring! See the [docs](osier.readthedocs.io)